apps/omg_eth/test/omg_eth/subscription_worker_test.exs

```diff
@@ -53,82 +53,4 @@ defmodule OMG.Eth.SubscriptionWorkerTest do
       end
     )
   end
-
-  test "that worker can subscribe to my own server via arguments" do
```

Comment:
I have removed the test because we've changed how we pass in `ws url` so the test has no relevance anymore.